openshift_scalability/config/ocp-logtest.yaml

```diff
@@ -0,0 +1,19 @@
+projects:
+  - num: 1
+    basename: logtest
+    ifexists: delete
+    tuning: default
+    templates:
+      - num: 1
+        file: ./content/logtest/logtest-rc.json
+        parameters:
+         - REPLICAS: "3"
```

Comment:
I'd expect the rest of the parameters to be exposed here, would you mind adding them to this file?

```--num-lines 0 --line-length 200 --word-length 9 --rate 60 --fixed-line...```